source/blockchain_details.rst

```diff
@@ -0,0 +1,431 @@
+.. _blockchain_details:
+
+
+Core blockchain details
+^^^^^^^^^^^^^^^^^^^^^^^
+
+As any blockchain - the Credits blockchain consists of Blocks. And every
+block contains transactions that happened since previous block created.
+
+Unlike other blockchains the Credits blockchain also contains States,
+which are snapshots of the contents of the blockchain created after each
+block.
```

Comment:
After each block what. is the problem, Its not clear. We should use the term application.